packages/web/src/common/store/pages/saved/lineups/sagas.ts

```diff
@@ -79,7 +81,9 @@ function* getTracks({ offset, limit }) {
   }
 
   if (allSavedTrackIds.length > 0) {
-    const tracks = yield call(retrieveTracks, {
+    // @ts-ignore - KJ: strings can be passed for the local save track ids
+    // Need to verify that this works like we expect
```

Comment:
does it work?

Reply:
also I bet we can change `ids: (ID | string)[]` to `ids: ID[] | string[]` in the lineup types